.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Docker Image in Docker Hub
+
+on:
+  push:
+    branches:
+      - main

```

Comment:
Si este path no incluye esta rama, ¿cómo se están publicando entonces las imágenes? ¿Por qué tenéis la manía de poner esto sólo porque vienen en el ejemplo?

Reply:
Pensaba aplicarlo solo cuando se hicieran cambios en la rama principal como cambios definitivos, pero tiene sentido incluir esta rama y luego quitarla para hacer tests en este objetivo